horizons/ai/aiplayer/mission/specialdomestictrade.py

```diff
@@ -114,8 +118,12 @@ def _reached_source_settlement(self):
 			self.report_failure('No resources to transport')
 
 	def _move_to_destination_settlement(self):
-		self._move_to_warehouse_area(self.destination_settlement_manager.settlement.warehouse.position, Callback(self._reached_destination_settlement),
```

Comment:
These changes make it unnecessary difficult to review changes. At the very least, I'd like to see these things broken up into 2 commits, format first, line length second.